R/qcMetric_MSMSScans.R

```diff
@@ -307,9 +307,14 @@ Heatmap score [MS<sup>2</sup> Scans: TopN ID over N]: Rewards uniform identifica
       df.ratio = ddply(DF, c("scan.event.number", "fc.raw.file"), function(x)
       {
         xp = xm = 0
-        if ("+" %in% x$identified) xp = x$n[x$identified=="+"]
-        if ("-" %in% x$identified) xm = x$n[x$identified=="-"]
-        ratio = xp * 100 / sum(xp, xm)
+        if(c("+","-") %in% x$identified){
+          if ("+" %in% x$identified) xp = x$n[x$identified=="+"]
+          if ("-" %in% x$identified) xm = x$n[x$identified=="-"]
+          ratio = xp * 100 / sum(xp, xm)} else{
+          if (1 %in% x$identified) xp = x$n[x$identified==1]
```

Comment:
this should be solved in `mztabreader::getMSMSScans()`, i.e. make the `$identified` column look like MaxQuants